platform/node/README.md

```diff
@@ -29,13 +29,25 @@ npm run test-suite
 ## Rendering a map tile
 
 ```js
+var fs = require('fs');
+var path = require('path');
 var mbgl = require('mapbox-gl-native');
 var sharp = require('sharp');
-var map = new mbgl.Map({ request: function() {} });
+
+var options = {
+  request: function(req, callback) {
+    fs.readFile(path.join(__dirname, 'test', req.url), function(err, data) {
+      callback(err, { data: data });
+    });
+  },
+  ratio: 1
+};
+
+var map = new mbgl.Map(options);
 
 map.load(require('./test/fixtures/style.json'));
 
-map.render({}, function(err, buffer) {
+map.render({zoom: 0}, function(err, buffer) {
```

Comment:
I think empty options are a little subtle to understand, so set a zoom should be OK.

I also noticed that a [debugOptions](https://github.com/mapbox/mapbox-gl-native/blob/08d0a9a0e2e771b87018bcb3d4d124c52fed1689/platform/node/src/node_map.cpp#L32) which is undocumented in node/readme.md. Does it available in node-mapbox-gl-native?